client/src/pages/Wallet.js

```diff
@@ -1,25 +1,114 @@
-import styled from 'styled-components';
+import styled from 'styled-components/macro';
+import PropTypes from 'prop-types';
 import Navigation from '../components/Navigation';
+import FavoriteCoinsCard from '../components/FavoriteCoins.js';
+import { useState, useEffect } from 'react';
+import { saveToLocal, loadFromLocal } from '../lib/localStorage';
+import WalletOverview from './WalletOverview';
+import AddForm from './AddForm';
+import Header from '../components/Header';
+
+export default function Wallet({
+  favoriteCoins,
+  exchanges,
+  walletOverview,
+  selectedCoin,
+  onSetWalletOverview,
+  onToggleFavoriteCoins,
+  onSetSelectedCoin,
+}) {
+  const [historicalCoins, setHistoricalCoins] = useState([]);
```

Comment:
I dont think you ever use historical Coins throughout your whole app, maybe its best to remove them if you - apparently - dont need them? :)

Reply:
removed :D